### PR TITLE
Add podcast post for Orta Therox

### PR DESCRIPTION
**Pull request by csbailey5t**
Includes:
- standard podcast template
- shownotes
- transcript

### CODE REVIEW

File: podcast/2.6.md [hunk 1]
```diff
@@ -0,0 +1,741 @@
...
+
+Apple definitely took about a decade to get on board with trying to improve that iteration cycle, but realistically, even their current approach, which is SwiftUI, it doesn't actually run your code and hot-reload it. It actually runs it separately in a sandbox that isn't your application, so it's not actually that accurate. It's just close enough. And so, you still don't get the React style. I press save and it does it all.
+
+**Beyang Liu:**
+Yeah. It's magical.
+
+**Orta Therox:**
+All those things were such compelling cases to React Native. There was just so much React advantage, and then there was also we got the advantage of using Relay, which is a really powerful way of connecting your API to your client. And we said that the apps I was working on, at Artsy, these were apps that were just pretty JSON parses, and native development SDKs, these are really built for building long-running applications like web browsers or your music player, messaging apps, not these quick, like, "Get me some JSON data, put it on a screen and then don't do anything until you click something next."
```
Comment (rebeccadee) [inline podcast/2.6.md:181]:
```suggestion
All those things were such compelling cases to React Native. There was just so much React advantage, and then there was also we got the advantage of using Relay, which is a really powerful way of connecting your API to your client. And we said that the apps I was working on, at Artsy, these were apps that were just pretty JSON parsers, and native development SDKs, these are really built for building long-running applications like web browsers or your music player, messaging apps, not these quick, like, "Get me some JSON data, put it on a screen and then don't do anything until you click something next."
```
@csbailey5t I didn't want to just change this, but was this a typo?

File: podcast/2.6.md [hunk 2]
```diff
@@ -0,0 +1,741 @@
...
+
+Adium: https://adium.im/
+
+TypeScript Playground: https://www.typescriptlang.org/play
+
+TypeScript Docs: https://www.typescriptlang.org/docs/
+
+Swift Playgrounds: https://www.apple.com/swift/playgrounds/
+
+Shiki Twoslash: https://github.com/shikijs/twoslash
```
Comment (rebeccadee) [inline podcast/2.6.md:71]: @csbailey5t The original link was 404ing, is this one suitable?

File: podcast/2.6.md [hunk 3]
```diff
@@ -0,0 +1,741 @@
...
+
+esbuild: https://esbuild.github.io/
+
+CocoaPods: https://cocoapods.org/
+
+TypeScript: https://www.typescriptlang.org/
+
+Flow: https://flow.org/
+
+Super Mario Maker: https://en.wikipedia.org/wiki/Super_Mario_Maker
```
Comment (rebeccadee) [inline podcast/2.6.md:57]: @jennybergen I think it's fine if there's no other obvious place to link

**Review by rebeccadee — COMMENTED** [review]
@csbailey5t @jennybergen I know there was discussion about adding a highlights section at the top, or doing things like pull quotes and subheads to break up the transcript. Should we be trying that out for this one?

File: podcast/2.6.md (hunk 2 above)
Comment (csbailey5t) [reply podcast/2.6.md:71]: There was a missing "s"; it should be https://shikijs.github.io/twoslash/ which is preferable to the github link

File: podcast/2.6.md (hunk 1 above)
Comment (csbailey5t) [reply podcast/2.6.md:181]: Yes, I think it's a typo after listening again.

File: podcast/2.6.md (hunk 2 above)
Comment (rebeccadee) [reply podcast/2.6.md:71]:
```suggestion
Shiki Twoslash: https://shikijs.github.io/twoslash/
```

**Comment by rebeccadee** [comment]
> I know there was discussion about adding a highlights section at the top, or doing things like pull quotes and subheads to break up the transcript. Should we be trying that out for this one?

Just closing the loop here, we will try this out for the next episode!

File: podcast/2.6.md [hunk 4]
```diff
@@ -0,0 +1,743 @@
...
+<span>
+Orta Therox, Beyang Liu
+</span>
+<!-- END GUESTS -->
+
+<!-- START SUMMARY -->
+
+How do you design software documentation and websites that both intrigue and educate? As a contributor to popular projects like React Native, Jest, Prettier, and TypeScript, Orta Therox has prioritized design for visual engagement, accessibility, and learning. In this episode, Orta talks about the importance of engaging docs, how experimentation fuels learning and engineering in TypeScript, and how developers can write better code examples with Shiki Twoslash, a project he developed and designed. Along the way, Orta also shares his own story of getting into code and the odd way he was hired on Microsoft's TypeScript compiler team.
+
+Click the audio player below to listen, or click [here]() to watch the video.
```
Comment (rebeccadee) [inline podcast/2.6.md:25]:
```suggestion
Click the audio player below to listen, or click [here](https://youtu.be/xGdSUUtu3i8) to watch the video.
```

**Pull request merged**